plugins/inputs/signalfx_metadata/signalfx-metadata.go

```diff
@@ -88,18 +92,20 @@ func (s *SFXMeta) sendNotifications(acc telegraf.Accumulator) {
 
 // Gather - read method for SignalFx metadata plugin
 func (s *SFXMeta) Gather(acc telegraf.Accumulator) error {
-	if s.processInfo == nil {
+	if s.processInfo == nil && s.ProcessInfo {
```

Comment:
Having two fields with one letter cased different is pretty tricky.  Does that even pass lint?

Reply:
Yes it does pass lint.  And in go it's the difference between something considered internal to the struct and external.  ProcessInfo must match the value in the config.